contracts/test/v0.8/dev/FunctionsDON.test.ts

```diff
@@ -69,8 +69,8 @@ describe('OCR2DROracle', () => {
       0,
       ethers.BigNumber.from(5021530000000000),
     )
-    oracle = await ocr2drOracleFactory.connect(roles.defaultAccount).deploy()
-    registry = await ocr2drRegistryFactory
+    oracle = await FunctionsOracleFactory.connect(roles.defaultAccount).deploy()
+    registry = await functionsRegistryFactory
```

Comment:
should this be renamed to `functionsBillingRegistryFactory` for consistency with the contract's name?

Reply:
Sure